core/src/main/java/hudson/model/Run.java

```diff
@@ -39,10 +39,7 @@
 import hudson.Util;
 import hudson.XmlFile;
 import hudson.cli.declarative.CLIMethod;
-import hudson.console.AnnotatedLargeText;
-import hudson.console.ConsoleLogFilter;
-import hudson.console.ConsoleNote;
-import hudson.console.ModelHyperlinkNote;
+import hudson.console.*;
```

Comment:
IDEA is, apparently. I set the number of imports to collapse into `*` to 15 (up from 5). I wonder whether this will introduce useless changes into the other direction.